docs/configuration/environment-variables.md

```diff
@@ -73,6 +73,19 @@ Boolean:
   files are served
 - `False` (default): the application is launched with debug mode turned off, similar to how it runs in production
 
+### `DJANGO_LOAD_SAMPLE_DATA`
```

Comment:
Thanks for adding this!

Can you also add a section somewhere (either in configuration or deployment?) about how to load new data in a different environment with a new `0002_migration.py` file?

Reply:
Also a mention of the `0003` migration and what it does.

Reply:
Probably in the new `configuration/data.md` docs from #814

Reply:
https://github.com/cal-itp/benefits/pull/827/commits/9aaa21ae20e77906217ec08c5d63a3ce23ce62c7 and https://github.com/cal-itp/benefits/pull/827/commits/0af40c3b52d046a5801ffcb869b407efacb12e2c